src/main/java/com/internet/shop/controller/InjectDataController.java

```diff
@@ -26,15 +28,17 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        userService.create(new User("User A", "userA", "1111"));
-        userService.create(new User("User B", "userB", "1111"));
-        userService.create(new User("User C", "userC", "1111"));
+        userService.create(new User("admin", "admin", "1111", Set.of(Role.of("ADMIN"))));
+        userService.create(new User("User A", "userA", "1111", Set.of(Role.of("USER"))));
+        userService.create(new User("User B", "userB", "1111", Set.of(Role.of("USER"))));
+        userService.create(new User("User C", "userC", "1111", Set.of(Role.of("USER"))));
         productService.create(new Product("BMW 320", 50_000));
         productService.create(new Product("BMW 530", 75_000));
         productService.create(new Product("BMW X5M50D", 120_000));
         shoppingCartService.create(new ShoppingCart(1L));
```

Comment:
let's not create a shopping cart for admin

Reply:
ok